src/pug/body.pug

```diff
@@ -0,0 +1,6 @@
+div.pf-c-background-image
+div.pf-c-page#page
+    include header.pug
+    include sidebar2.pug
+    main.pf-c-page__main#root(role='main')
```

Comment:
We are missing `include footer.pug` here.

Reply:
It sounds like we're trying to kill `footer.pug` though. @iphands ?

Reply:
Yeah. Kill footer.
For the longest time it was embedded in <main> so it would disappear as soon as the page loaded anyway. Because of <- no one can rightly claim they were using it for anything. Also for a few weeks now footer has been an empty string.